README.md

```diff
@@ -113,8 +113,17 @@ $ kubectl get packagemanifest container-security-operator
 
 ### Using kubectl
 
-Check if a pod has any vulnerability, and list the CVEs, if any:
+Get a list of all the pods affected by vulnerable images detected by the Operator:
+```sh
+$ kubectl get imagemanifestvuln --all-namespaces -o json | jq '.items[].status.affectedPods'
```

Comment:
`kubectl get imagemanifestvuln --all-namespaces -o json | jq '.items[].status.affectedPods' | jq 'keys' | jq 'unique'`
will only list names and remove duplicates.